packages/utils-algebra/lib/utils.ts

```diff
@@ -78,18 +79,66 @@ const transformer = new TransformerSubTyped<KnownOperation>({
   [Types.COPY]: { ignoreKeys: new Set([ 'source', 'destination', 'metadata' ]) },
 });
 
+/**
+ * Transform a single operation.
+ * The transformation calls the preVisitor starting from the startObject.
+ * The preVisitor can dictate whether transformation should be stopped.
+ * Note that stopping the transformation also prevets further copying.
+ * The transformer itself transforms operations starting with the deepest one that can be visited.
+ * The transformer callback is performed on a copy of the original.
+ * @param startObject
+ * @param nodeCallBacks
+ */
 export const mapOperation: (typeof transformer.transformNode<'unsafe', Operation>) = <any>
```

Comment:
Given the importance of this function, I'd consider including a small example in the jsdocs.